source/getting-started/automation-action.markdown

```diff
@@ -51,6 +51,7 @@ automation 2:
 
 Conditions can also be part of an action:
 
+{% raw %}
 ```yaml
```

Comment:
`{% raw %}` needs to be inside the YAML block. I guess that's my fault because I told you to do it that way last time.